src/components/MainMenu.jsx

```diff
@@ -1,18 +1,87 @@
 import styled from "styled-components";
 import { Routes, Route, NavLink, Link } from "react-router-dom";
+import logo from "../images/Logo_nacHHaltig.png";
+import NavLinks from "./NavLinks";
+
+import Restaurants from "../pages/Restaurants";
+import Shopping from "../pages/Shopping";
+import Beauty from "../pages/Beauty";
+import Favourites from "../pages/Favourites";
+import Calculator from "../pages/Calculator";
+import NotFound from "../pages/NotFound";
 
 function MainMenu() {
   return (
     <>
-      <nav>
-        <NavLink to="Restaurants">Restaurants</NavLink> <br />
-        <NavLink to="Shopping">Shopping</NavLink> <br />
-        <NavLink to="Beauty + Hair">Beauty + Hair</NavLink> <br />
-        <NavLink to="Favoriten">Favoriten</NavLink> <br />
-        <NavLink to="Co2-Rechner">Co2-Rechner</NavLink>
-      </nav>
+      <Header>
+        <Link to="/">
+          <LogoImage src={logo} alt="Logo" />
+        </Link>
+      </Header>
+
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <>
+              <LogoImage />
+              <NavContainer>
+                <NavLinks />
+              </NavContainer>
+            </>
+          }
+        />
```

Comment:
```suggestion
          element={<MainMenu />}
        />
```
Would that be the MainMenu? Consider extracting this part in one page/component.

Reply:
I've extracted it, you can see it after the next push :)